.all-contributorsrc

```diff
@@ -1816,6 +1816,15 @@
       "contributions": [
         "translation"
       ]
+    },
```

Comment:
hi @banditelol i already added you on a separate pull request as a contributor :)
could you please remove this file from your pull request,
so that your pull request only includes the modifications to the file src/math/p5.Vector.js
thank you!

Reply:
I've removed as per your request